### PR TITLE
Fixed chart to display times shown and times clicked

### PR DESCRIPTION
**Pull request by RecursiveIteration**
#17 

Chart is fixed!

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -71,8 +71,15 @@ function printReport() {
     data: {
       labels: getNames(),
       datasets: [{
-        label: 'Popularity',
-        data: getPopularity(),
+        label: 'Times Clicked',
+        data: getPopularity()[0],
+        backgroundColor: '#8f1386',
```
Comment (dmuldrew) [inline js/app.js:76]: I love color #8f1386!!!

File: js/app.js [hunk 2]
```diff
@@ -100,16 +107,13 @@ function getNames () {
 }
 
 function getPopularity () {
-  var result = [];
+  var numClicks = [];
+  var numShows = [];
   for (var i in gameState.productMap) {
-    try {
-      var percent = gameState.productMap[i].numberOfClicks / gameState.productMap[i].numberOfShows;
```
Comment (dmuldrew) [inline js/app.js:106]: Looks like calculating percent had some issue?

File: js/app.js [hunk 3]
```diff
@@ -71,8 +71,15 @@ function printReport() {
     data: {
       labels: getNames(),
       datasets: [{
-        label: 'Popularity',
-        data: getPopularity(),
+        label: 'Times Clicked',
+        data: getPopularity()[0],
```
Comment (dmuldrew) [inline js/app.js:75]: maybe a bit clearer to be able to say 'getPopularity()['numClicks']' by returning ' {'numClicks': numClicks, 'numShows': numShows}'

File: js/app.js [hunk 4]
```diff
@@ -71,8 +71,15 @@ function printReport() {
...
+        backgroundColor: '#8f1386',
+        borderColor: '#8f1386',
+        borderWidth: 1
+      },
+      {
+        label: 'Times Shown',
+        data: getPopularity()[1],
```
Comment (dmuldrew) [inline js/app.js:82]: like the function call for the data property

File: index.html [hunk 5]
```diff
@@ -13,7 +13,7 @@
     <hr />
     <div id="info">
       <p id="intro">Welcome to the <span>Bus-Mall</span>&trade; product popularity application. We are working to improve our magazine and we could use your help! Aid us in selecting the products we feature in our puplication by choosing the items that interest you most. For each set of products we present just click on the item that most strikes your fancy.  Thanks for your assistance!</p>
-      <p id="close">Thank you for participating! The following chart summarizes your choices. Results represent the fraction of times you selected an item divided by the number of times that item was shown to you.</p>
```
Comment (qlalberta) [inline index.html:16]: good job to be consistent with your change of results.

**Review by dmuldrew — COMMENTED** [review]
nice easy to follow changes

**Pull request merged**